t04_refactored.py

```diff
@@ -153,8 +153,40 @@ def team_6_adv():
 ###################################################################################
 
 def team_7_adv():
-    pass
-    # TODO Add your code here
+    """"
+    https://docs.google.com/document/d/16k30UI8SAnGJHR-KdZ2CAgtkM6OhWY8SjExevrOug18/edit?usp=sharing
+    Aaliyah Murphy
+    Abdul-Hakim Adam
+    """
+    # TEAM 7
+    global isDead
+    direction = input("Go with him further [Up, or Down]?")
+    if direction == "Up":
+        print("You've reached a large vault... a large console stands to its right...")
+        print("The console lights up when you reach it... it asks for a passcode")
+        user_input = input("Guess the code/Ask for help")
+        if user_input == "Ask for help":
+            print("You ask Dr. Heggen for help")
+            print("He pulls out a small electrical device and plugs it into the console")
+            print("The console glows green and the vault opens up")
+            print("you've found riches beyond your wildest dreams")
+        elif user_input == "Guess the code":
+            print("You type in '12345' and the console turns red")
+            print("The walls close in on you, crushing you like a Wookie in a trash compactor")
+            isDead = True
+
+    elif direction == "Down":
+        print("you have reached the strongest creature in all of the lands..")
+        sleep(delay * 3)
+        print("You tried to run...")
+        isDead = True
+    # TODO Make sure to add the additional check if the user makes the "bad" choice!
+    else:
+        print("You tried to run...")
+        isDead = True
+        kill_if_dead(isDead)
```

Comment:
This only gets called in the else: clause. It should probably be called outside of the if/else chain.